github/issues_comments.go

```diff
@@ -100,8 +103,13 @@ func (s *IssuesService) GetComment(owner string, repo string, id int) (*IssueCom
 
 // CreateComment creates a new comment on the specified issue.
 //
+// Comments longer than 65536 will return an error.
+//
 // GitHub API docs: http://developer.github.com/v3/issues/comments/#create-a-comment
 func (s *IssuesService) CreateComment(owner string, repo string, number int, comment *IssueComment) (*IssueComment, *Response, error) {
+	if comment != nil && comment.Body != nil && len(*comment.Body) > maxCommentLen {
+		return nil, nil, fmt.Errorf("Comments may not be longer than %d", maxCommentLen)
+	}
 	u := fmt.Sprintf("repos/%v/%v/issues/%d/comments", owner, repo, number)
```

Comment:
Add a blank line before `u := ...` to separate it, like you've done in `IssuesService.Create`. The local validation is a separate section that has nothing to do with this code.